### PR TITLE
Unused import

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by rouazana** [comment]
:+1:

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

File: protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 
 package org.apache.james.protocols.smtp;
+
+import java.util.List;
```
Comment (aduprat) [inline protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java]: MINOR Remove this unused import 'java.util.List'. [rule](http://62.210.101.42:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by aduprat** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by aduprat** [comment]
test this please

File: protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java (hunk 1 above)
Comment (rouazana) [inline protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java]: \o/

Comment (aduprat) [inline protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java]: MINOR Remove this unused import 'java.util.List'. [rule](http://172.17.0.2:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: server/mailet/mailetcontainer-api/src/main/java/org/apache/james/mailetcontainer/api/MailetLoader.java [hunk 2]
```diff
@@ -22,6 +22,8 @@
 import org.apache.mailet.Mailet;
 import org.apache.mailet.MailetConfig;
 
+import java.util.List;
```
Comment (aduprat) [inline server/mailet/mailetcontainer-api/src/main/java/org/apache/james/mailetcontainer/api/MailetLoader.java]: MINOR Remove this unused import 'java.util.List'. [rule](http://172.17.0.2:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by aduprat** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by aduprat** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

File: protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java (hunk 1 above)
Comment (aduprat) [inline protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java]: MINOR Remove this unused import 'java.util.List'. [rule](http://172.17.0.2:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

File: server/mailet/mailetcontainer-api/src/main/java/org/apache/james/mailetcontainer/api/MailetLoader.java (hunk 2 above)
Comment (aduprat) [inline server/mailet/mailetcontainer-api/src/main/java/org/apache/james/mailetcontainer/api/MailetLoader.java]: MINOR Remove this unused import 'java.util.List'. [rule](http://172.17.0.2:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by aduprat** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Pull request closed without merging**